src/components/controls/TextContainer.tsx

```diff
@@ -7,6 +7,7 @@ interface IInputProps {
     label: string | JSX.Element;
     value?: string | number | null | Date;
     darkerInputLabel?: boolean;
+    disabled?: boolean;
```

Comment:
Why does a TextContainer that is never editable, a prop named disabled?

Reply:
Now TextContainer has always static height